<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>info</class>
 <widget class="QDialog" name="info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>298</width>
      <height>198</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>9</horstretch>
       <verstretch>14</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>USER</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Calligraphy</family>
      </font>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Calligraphy</family>
      </font>
     </property>
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Calligraphy</family>
      </font>
     </property>
     <property name="text">
      <string>SSN:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Calligraphy</family>
      </font>
     </property>
     <property name="text">
      <string>Job:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Calligraphy</family>
      </font>
     </property>
     <property name="text">
      <string>Email:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="NameUser">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Job">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Email">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="IDUser">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="SSNUser">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>300</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>298</width>
      <height>198</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ACCOUNT</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Calligraphy</family>
      </font>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Calligraphy</family>
      </font>
     </property>
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Calligraphy</family>
      </font>
     </property>
     <property name="text">
      <string>Reader:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Calligraphy</family>
      </font>
     </property>
     <property name="text">
      <string>Admin</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Calligraphy</family>
      </font>
     </property>
     <property name="text">
      <string>Blirarian:</string>
     </property>
    </widget>
    <widget class="QLabel" name="AccountName">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="AccountID">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="IsReader">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="IsBlirarian">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 0);</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="IsAdmin">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>169</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 0);</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea_3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>0</y>
     <width>300</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>298</width>
      <height>198</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>PASSWORD</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>New password:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Comfirm password:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="OldPass">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="ComfirmPass">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="NewPass">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="ChangeInfo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string>Change info</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ChangePass">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>200</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Change password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Ok">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea_4">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>0</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>298</width>
      <height>298</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>HISTORY</string>
     </property>
    </widget>
    <widget class="QLabel" name="History5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>300</width>
       <height>200</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">alternate-background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Huy</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
